### ISSUE TITLE
Beta text in header fails color contrast according to lighthouse

### ISSUE THREAD

**Issue by aaronhans** [cagov/design-system#434]
Running lighthouse from web.dev/measure against the site homepage catches the Beta text in the site header as failing color contrast

**Comment by aaronhans** [comment]
I see #ff800 in figma so we are matching current design. @adam-little does this pass color contrast checking that you do in your design tools?

**Comment by aaronhans** [comment]
Assigning to Adam to check color contrast in design tools

**Comment by adam-little** [comment]
That's correct, the Secondary color (FF8000) does not pass accessibility when on a white background. Let's use Secondary-Dark (B35A00).

**Comment by aaronhans** [comment]
Fix in https://github.com/cagov/design-system/pull/614

**Comment by jaredwyee** [comment]
Confirmed B35A00 with eye dropper on https://1-0-1-rc-post-launch-fixes.pr.designsystem.webstandards.ca.gov/
![image](https://user-images.githubusercontent.com/85529382/153266052-91cac838-d474-41fe-a347-bd4ce9dafe65.png)
